src/setViewport.js

```diff
@@ -1,11 +1,12 @@
+import getDefaultViewport from './internal/getDefaultViewport.js';
 import { getEnabledElement } from './enabledElements.js';
 import updateImage from './updateImage.js';
 
 const MIN_WINDOW_WIDTH = 0.000001;
 const MIN_VIEWPORT_SCALE = 0.0001;
 
 /**
- * Sets the viewport for an element and corrects invalid values
+ * Sets/updates viewport of a given enabled element
  *
  * @param {HTMLElement} element - DOM element of the enabled element
  * @param {Viewport} viewport - Object containing the viewport properties
```

Comment:
If Viewport is now optional, could you update the jsdoc to reflect this? You just need square brackets around the parameter name.

e.g. * @param {Viewport} [viewport] - ...